.applier/group_vars/seed-hosts.yml

```diff
@@ -66,6 +66,12 @@ test_pipelines:
       PIPELINE_FILENAME: Jenkinsfile.test
       PIPELINE_SOURCE_REPOSITORY_REF: "{{ slave_repo_ref }}"
       PIPELINE_SOURCE_REPOSITORY_URL: "{{ repository_url }}"      
+    argocd:
```

Comment:
I'm assuming this should be `helm`

Reply:
appreciate the review, fixed up in the next commit